### PR TITLE
(DONOTMERGE) Extend timeout to 5m

### PR DESCRIPTION
**Pull request by michaeltlombardi**

### CODE REVIEW

File: lib/puppet/provider/exec/powershell.rb [hunk 1]
```diff
@@ -47,7 +47,7 @@ def self.upgrade_message
 
   def ps_manager
     debug_output = Puppet::Util::Log.level == :debug
-    Pwsh::Manager.instance(Pwsh::Manager.powershell_path, Pwsh::Manager.powershell_args, debug: debug_output)
+    Pwsh::Manager.instance(Pwsh::Manager.powershell_path, Pwsh::Manager.powershell_args, debug: debug_output, pipe_timeout: 300)
```
Comment (joshcooper) [inline lib/puppet/provider/exec/powershell.rb:50]: what was the timeout previously?

Comment (pillarsdotnet) [reply lib/puppet/provider/exec/powershell.rb:50]: It defaults to 30 here: https://github.com/puppetlabs/ruby-pwsh/blob/main/lib/pwsh.rb#L37

**Review by binford2k — APPROVED** [review]
lgtm.

We can talk about adding a generic scripting host timeout to exec another time. And the possibility of extending the resource API to powershell providers is a thing we should consider too. But neither of those needs to wait on this.

**Comment by michaeltlombardi** [comment]
Closed in favor of #321

**Pull request closed without merging**